src/MarginTrading.DataReader/Controllers/TradeMonitoringController.cs

```diff
@@ -86,10 +90,12 @@ public class TradeMonitoringController : Controller
         /// <response code="200">Returns opened positions</response>
         [HttpGet]
         [Route("openPositions/byVolume/{volume}")]
-        public async Task<List<OrderContract>> GetOpenPositionsByVolume([FromRoute]decimal volume)
+        public async Task<List<OrderContract>> OpenPositionsByVolume([FromRoute]decimal volume)
         {
             return (await _ordersSnapshotReaderService.GetActiveAsync())
-                .Where(order => order.GetMatchedVolume() >= volume).Select(OrderExtensions.ToBaseContract)
+                .Where(order => order.GetMatchedVolume() >= volume)
+                .Select(OrderExtensions.ToBaseContract)
```

Comment:
Why not to use direct conversion instead of 2 subsequent ?

Reply:
Had the same question ))) Let's leave it as is, I will fix such cases later. Added to TODO )